test/jackdaw/serdes/edn_test.clj

```diff
@@ -18,3 +19,15 @@
         (is (= (seq bytes)
                (seq (->> (.deserialize (jse/deserializer) nil bytes)
                          (.serialize (jse/serializer) nil)))))))))
+
+(defmethod print-method java.net.URI
```

Comment:
Since this defmethod unfortunately goes into global scope, I'd rather we use `#jackdaw/uri` since the ns unqualified space is nominally reserved for future additions from core.

Reply:
@arrdem done